### ISSUE TITLE
where is the contents of "model2"?

### ISSUE THREAD

**Issue by shengyong1** [Cloudslab/DLSF#1]
please quickly answer,thank you.

**Comment by shreshthtuli** [comment]
Please follow instructions mentioned in the readme. Once you run the code, the pytorch models would be saved in _CloudSim/model2_ directory.

**Comment by shengyong1** [comment]
> Please follow instructions mentioned in the readme. Once you run the code, the pytorch models would be saved in _CloudSim/model2_ directory.

Thank you for your answer! And how do you combine Java files and python files? I can't find the relevant code?

**Comment by shreshthtuli** [comment]
All java-python communication code can be found here: https://github.com/Cloudslab/DLSF/tree/master/CloudSim/modules/cloudsim-examples/src/main/java/org/cloudbus/cloudsim/examples/power/DeepRL

**Comment by shengyong1** [comment]
> All java-python communication code can be found here: https://github.com/Cloudslab/DLSF/tree/master/CloudSim/modules/cloudsim-examples/src/main/java/org/cloudbus/cloudsim/examples/power/DeepRL

I have another question，I follow your instructions mentioned in the readme.But some console input is required. What do these consoles need to input? What does setInput backprop and so on in the DeepRL.py file mean? Thank you for your answer

**Comment by shreshthtuli** [comment]
You dont need to enter anything on the console. All communication is done via the java python binding.

**Comment by shengyong1** [comment]
> You dont need to enter anything on the console. All communication is done via the java python binding.

I just need to run DeepRL.py? But I run this way without results.

**Comment by shreshthtuli** [comment]
Just run it on a terminal and run CloudSim. Resutls would be generated in _CloudSim/model2/_ directory.

**Comment by shengyong1** [comment]
> 只需在终端上运行它并运行CloudSim。结果将在_CloudSim / model2 /_目录中生成。

how to run CloudSim? run DeepRLRunner.java?

**Comment by shreshthtuli** [comment]
Yes.

**Comment by shengyong1** [comment]
I can't run for result! I run DeepRLRunner.java and DeepRL.py for a result with this:
Ran python code : C:\Users\shengyong\Desktop\DLSF-master\Deep-Learning\DeepRL.py
Creating VMs...
99%
DL Loss = null
0 hr 5 min - 19
Could you please check the code on your GitHub? Are there any gaps?

**Comment by shengyong1** [comment]
> Yes.
I can't run for result! I run DeepRLRunner.java and DeepRL.py for a result with this:
Ran python code : C:\Users\shengyong\Desktop\DLSF-master\Deep-Learning\DeepRL.py
Creating VMs...
99%
DL Loss = null
0 hr 5 min - 19
Could you please check the code on your GitHub? Are there any gaps?

**Comment by shengyong1** [comment]
> 是。

I'll give it to you DRLDatacenter.java The update dlmodel function break point in the file does not run to this function. Why is there such a result?

**Comment by shreshthtuli** [comment]
I can not help you on your specific code.
